src/restaurant_inspections_api/services.clj

```diff
@@ -84,11 +84,11 @@
   (db/select-counties-summary))
 
 (defn- format-businesses-params
-  [params-map]
-  (assoc params-map
-         :zipCodes (when-let [zipCodes (:zipCodes params-map)]
-                     (str/split zipCodes #","))
-         :page (* (:page params-map) (:perPage params-map))))
+  [{:keys [zipCodes perPage page countyNumber]}]
```

Comment:
Checking the history. Back when I first tried this I was confused by the keys and symbols used : `camelCase` in some cases and `dashed-names`. It even break at some point. But now I just pass the context around and it used the `camelCase` keys all the time